spec.md

```diff
@@ -0,0 +1,232 @@
+1. **Clearing House for [Well-known Artifacts][def-well-known-types]** - artifact authors can submit their artifact definitions, providing registry operators a list by which they can easily support.
+
+[Artifacts v1.0][artifacts] presumes a refactoring of the [OCI Image-spec 1.0][image-spec] and the pending [OCI Distribution-spec 1.0][distribution-spec] to refactor [manifest][image-manifest] and [index][image-index] to separately versioned specs. The refactoring is for clarity of use, and is not required to support artifacts.
+
+**Reference**:
+
+- [Need a Working Group for Updating the distribution-spec #9](https://github.com/opencontainers/artifacts/issues/9)
+- [Need a Working Group for Updating the image-spec #10](https://github.com/opencontainers/artifacts/issues/10)
+
+**v1.0 of the image-spec** defines index and manifest.
+
+![image-spec](./media/image-spec-objects.png)
+
+**Artifacts 1.0** visualizes a refactoring of index and manifest.
+
+![refactored artifacts spec](./media/artifacts-spec-objects.png)
+
+## Table of Contents
+
+- [Scope](#scope)
+- [Defining OCI Artifact Types](#defining-oci-artifact-types)
+- [Defining a Unique Artifact Type](#defining-a-unique-artifact-type)
+- [Defining Supported Layer Types](#defining-supported-layer-types)
+- [Optional: Defining Config Schema](#optional-defining-config-schema)
+- [References](#references)
+
+## Scope
+
+[Artifacts v1.0][artifacts] is dependent on [OCI manifest][image-manifest] 1.0 to represent individual artifacts including [OCI Image][image-spec], [Helm][helm] and [Singularity][singularity].
+
+### Future Scope
+
+Future versions will support new artifact types that represent collections of artifacts using [OCI Index][image-index]. A means to identify an index as a type of artifact will be required.
+
+## Defining OCI Artifact Types
+
+As registries become content addressable distribution points, tools that pull artifacts must know if they can operate on the artifact. Artifact types are equivalent to file extensions.
+
+- When users open files, the host operating system typically launches the appropriate program.
+- When users open a file, from within a program, the open dialog filters to the supported types.
+- When search or security software scan the contents of a storage solution, the software must to know how to process the different types of content.
+- When users view the contents of a storage solution, they see the textual and visual indications of the type.
+
+OCI Artifacts provides these core capabilities to [OCI distribution spec][distribution-spec] based registries.
+
+### Visualizing Artifacts
+
+The manifest `config.mediaType` is the equivalent of a file extension, enabling artifact type differentiation.
+
+Once complete, artifacts can be identified and visualized as the following:
```

Comment:
s/once complete//

Reply:
following media type:

Reply:
media vs.. content / .. suggest picking one and search replace .. probably use stored media.. then media for short as you go along .. maybe say what media is.. a block or blocks of storage representing a file or stream of media.. yada yada.